src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+
+      {
+        todos.map(todo => (
+          <TodoItem key={todo.id} todo={todo} />
+        ))
+      }
+
+      {/* This todo is being edited */}
+      {false && (
+        <div className="todo">
+          <label className="todo__status-label">
+            <input
+              type="checkbox"
+              className="todo__status"
+            />
+          </label>
+
+          {/* This form is shown instead of the title and remove button */}
+          <form>
+            <input
+              type="text"
+              className="todo__title-field"
+              placeholder="Empty todo will be deleted"
+            // value="Todo is being edited now"
+            />
+          </form>
+
+          <div className="modal overlay">
+            <div className="modal-background has-background-white-ter" />
+            <div className="loader" />
+          </div>
+        </div>
+      )}
+
+      {/* This todo is in loadind state */}
+      {false && (
```

Comment:
Replace the hardcoded `false` with a state variable to toggle the 'loading' state of a todo.

Reply:
Done